Jenkinsfile.win

```diff
@@ -32,6 +32,8 @@ node(WIN_BUILD_NODE) {
 
     def helpers = load "artifacts/build-helpers.groovy"
 
+    sh "find .vagrant/machines/ -type d -name '*.vmx.lck' -print0 | xargs -0 rm -rf"
```

Comment:
can you please add some comment, why do we need this?

Reply:
on Win64 builder this file stay in system after "vagrant halt --force" command and bolck launching next build